src/toast/toast.styles.tsx

```diff
@@ -136,6 +138,7 @@ export const ActionButton = styled(Button.Small)`
 export const DismissButton = styled(ClickableIcon)<StyleProps>`
     padding: 0.75rem;
     margin: -0.75rem;
+    align-items: stretch;
```

Comment:
`align-items` should be applied to the flex container, which in this case is `Wrapper`

`align-self` is used for individual flex children